domain/src/main/kotlin/no/nav/su/se/bakover/domain/revurdering/VurderOpphørVedRevurdering.kt

```diff
@@ -10,15 +10,25 @@ import no.nav.su.se.bakover.domain.vilkår.Vilkårsvurderinger
 import java.time.Clock
 import java.time.LocalDate
 
+sealed class BeregningEllerRevurderingsPeriode {
```

Comment:
Jeg er ikke helt sikker på om jeg forstår helt hva denne klassen gjør? Bare en potet-wrapper for alle tilstander som ikke har en beregning?

Reply:
I typescript ville den vært en Beregning | Revurderingsperiode. Som betyr at vi ønsker å ta inn en Beregning eller en periode. Men man kan jo argumentere for at man kun vel bruke perioden fra revurderingsperioden og ikke bruke den i beregningen (evt. sjekke at de henger sammen).

Reply:
Når det er sagt, så begynner vel denne og bli ganske rotete. Kanskje hver Revurderings subclass burde hatt hver sin sjekk?